bootstrap/terraform/gcp-bootstrap/main.tf

```diff
@@ -40,10 +40,17 @@ module "gke" {
   http_load_balancing        = true
   remove_default_node_pool   = true
   add_cluster_firewall_rules = true
-  network_policy             = true
+  network_policy             = false
```

Comment:
what's the implication of setting this to false?  Is it duplicative with cilium?